### ISSUE TITLE
Updating function names

### ISSUE THREAD

**Issue by esganzerla** [meteor-factory/react-native-tinder-swipe-cards#72]
Hello,

I made a few pull requests because I'm using this component on a project and thought that we could set a better organization by defining some function names differently.

Some suggestions:
- `renderYup` to `renderSwipedRight`
- `renderNope` to `renderSwipedLeft`
- `renderMaybe` to `renderSwipedUp` (based on a PR #70)

- `handleYup` to `onSwipeRight`
- `handleNope` to `onSwipeLeft`
- `handleMaybe` to `onSwipeUp` (based on a PR #70)

- `yup` to `swipeRight` (based on PR #71)
- `nope` to `swipeLeft` (based on PR #71)
- `maybe` to `swipeUp` (based on PR #70 and #71)

- `cardRemoved` to `onCardRemoved`
- `renderNoMoreCards` to `renderEmpty`

I didn't take too much time to think about it because I believe I just first talk to the contributors to just make it better. I guess after some discussion we might get to some more conclusions like `yupStyle` and `yupTextStyle`.

Another suggestion would be to group all styles props to one or two props and nested categories inside to keep the component cleaner when including in a project.

I thinks this changes would be nice to make the component more generic and consistent.
I also think that there should be some fallbacks for those users that were using before updates.

Looking foward to know your opinions on that.

**Comment by esganzerla** [comment]
By the way, I forgot to say...

I'm using the `maybe` option (#70) because on my project it's exactly that name, but the new swipe up action is actually present on Tinder now as super like. So I guess any action might have a different purpose for each project.

**Comment by neyaoz** [comment]
@esganzerla I agree with you. We are using this plugin to present a list in swipe card format. For example, "Maybe" direction is going to be used as details, and it is not semantic. It is an amazing plugin, let's make it universal.
